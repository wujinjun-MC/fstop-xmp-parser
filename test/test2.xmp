<?xpacket begin="﻿" id="W5M0MpCehiHzreSzNTczkc9d"?>
<x:xmpmeta xmlns:x="adobe:ns:meta/" x:xmptk="XMP Core 4.4.0-Exiv2">
 <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
  <rdf:Description rdf:about=""
    xmlns:dc="http://purl.org/dc/elements/1.1/"
    xmlns:fstop="http://www.fstopapp.com/xmp/"
   fstop:favorite="0">
   <dc:subject>
    <rdf:Bag>
     <rdf:li>中文测试</rdf:li>
     <rdf:li>0.0</rdf:li>
     <rdf:li>wujinjun-MC</rdf:li>
    </rdf:Bag>
   </dc:subject>
  </rdf:Description>
 </rdf:RDF>
</x:xmpmeta>
<?xpacket end="w"?>
