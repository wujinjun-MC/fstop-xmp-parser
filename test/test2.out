{"test/test2.xmp":{"tags":["中文测试","0.0","wujinjun-MC"]}}
